code/modules/mob/living/carbon/xenomorph/abilities/general_powers.dm

```diff
@@ -985,7 +985,11 @@
 
 	if(blunt_stab)
 		stabbing_xeno.visible_message(SPAN_XENOWARNING("\The [stabbing_xeno] swipes its tail into [target]'s [limb ? limb.display_name : "chest"], bashing it!"), SPAN_XENOWARNING("You swipe your tail into [target]'s [limb? limb.display_name : "chest"], bashing it!"))
-		playsound(target, "punch", 50, TRUE)
+		var/playing_sound = rand(1, 99)
+		if(playing_sound == 99)
+			playsound(target, "bonk", 50, TRUE)
```

Comment:
"punch" is not a real sound, it just leads to a get_sfx switch where
`			if("punch")
S = pick('sound/weapons/punch1.ogg','sound/weapons/punch2.ogg','sound/weapons/punch3.ogg','sound/weapons/punch4.ogg')`

Reply:
i forgor , i just fixed it